roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -381,7 +381,6 @@ openstack_lbaas_monitor_max_retries: "3"
 openstack_cacert: "{{ lookup('env','OS_CACERT') }}"
 
 # Default values for the external OpenStack Cloud Controller
-external_openstack_lbaas_use_octavia: true
```

Comment:
I just guessed here can be
```
external_openstack_lbaas_use_octavia: false
```
and to understand the default value easily and the upper line can be like
```
use-octavia={{ external_openstack_lbaas_use_octavia }}
```
without embedding the default value into roles.

Reply:
please look last commit